src/tests/build_test.ts

```diff
@@ -48,6 +48,21 @@ Deno.test("build hook tests", async (t) => {
                 "properties": {},
               },
             },
+            "api_function_that_should_not_be_built": {
+              "type": "API",
+              "title": "API function",
+              "description": "should most definitely not be bundled",
+              "source_file":
+                "src/tests/fixtures/functions/this_shouldnt_matter.ts",
```

Comment:
From what I understand API functions should have this `source_file` field, I think `builtins` should we allow this field to be `undefined`

Also this might involve some refactoring but should we treat `API` and `builtin` type similarly?

Reply:
Good question, I'm not sure on the exact manifest schema for API and builtin functions! But, both, IIUC, do not rely on custom deno function code, so I don't think the `build` hook, which bundles custom function code, applies to them?

This patch fix was mainly for some internal process (that I'm unfamiliar with) using the `build` hook as part of creating API functions 🤷

Reply:
> Also this might involve some refactoring but should we treat API and builtin type similarly?

We don't need to because a built-in isn't ever added to the manifest

Reply:
> From what I understand API functions should have this source_file field

I've heard that they might eventually, but we did not have that support in the old builder. We're going to be working on supporting API functions more in depth this quarter, so we'll modify this logic then if need be.

Reply:
> From what I understand API functions should have this source_file field

I've heard that they might eventually, but we did not have that support in the old builder. We're going to be working on supporting API functions more in depth this quarter, so we'll modify this logic then if need be.

Reply:
> We don't need to because a built-in isn't ever added to the manifest

@shapirone when I did some prototyping I ended up changing the [workflow-step implementation](https://github.com/slackapi/deno-slack-sdk/blob/main/src/workflows/workflow-step.ts#L39) so that "API function" are not included similarly to how builtins behave, maybe this is a behavior to look into for next quarter